### PR TITLE
Lzi  revamp profile organization page

### PR DESCRIPTION
**Pull request by ZzZimm**
- Changing the UI/UX for this page (see pics below)
- Create a MutableField component: it's possible to use it on any form using a array of fields with field name, field title and base value. Base value is useful when cancelling the modification
Screenshot 2020-04-30 at 12 32 48
Screenshot 2020-04-30 at 12 32 53

### CODE REVIEW

File: src/components/user/MutableField.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+type Props = {
+  baseValue: string;
+  inputData: any;
+  title: string;
+  fieldName: string;
+  onChangeData: (fieldName: string, e: any) => void;
+  onSubmit: (organization: Organization) => Promise<any>;
+};
+
+function UserProfile(props: Props) {
```
Comment (ComeTiss) [inline src/components/user/MutableField.tsx]: MutableField*

File: src/service/models/user.model.tsx [hunk 2]
```diff
@@ -10,20 +11,20 @@ class User {
...
-    this.contactID = props.contactID;
+    this.contacts = new Contacts(props.contacts);
     this.country = props.country;
     this.email = props.email;
     this.firstName = props.firstName;
     this.functionTitle = props.functionTitle;
     this.isAdmin = props.isAdmin;
     this.lastName = props.lastName;
-    this.organizationId = props.organizationId;
+    this.organization = new Organization(props.organization);
```
Comment (ComeTiss) [inline src/service/models/user.model.tsx:27]: Can you not just get the organization.id?

**Comment by ComeTiss** [comment]
Nice one!

File: src/components/user/MutableField.tsx (hunk 1 above)
Comment (ZzZimm) [reply src/components/user/MutableField.tsx]: Good catch

File: src/service/models/user.model.tsx (hunk 2 above)
Comment (ZzZimm) [reply src/service/models/user.model.tsx:27]: Hmm, I'm not sure to get your point. When we fetch a user, we also fetch its associated organization (see gql schema)

**Review by LuciusA — APPROVED** [review]
Nice !

**Pull request merged**